### PR TITLE
chore: release 13.5.0 - update changelog

### PR DESCRIPTION
**Pull request by emilyrohrbough**
- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by emilyrohrbough** [comment]
Changelog PR: https://github.com/cypress-io/cypress-documentation/pull/5556

**Review by chrisbreiding — APPROVED** [review]

**Review by jennifer-shehane — APPROVED** [review]

File: .circleci/workflows.yml [hunk 1]
```diff
@@ -1342,18 +1342,7 @@ jobs:
...
-          steps: # issue with ssh keys on windows
-            - checkout
-      - unless:
-          condition:
-            equal: [ *windows-executor, << parameters.executor >> ]
-          steps:
-            - run:
-                name: checkout
-                command: git clone -b "$CIRCLE_BRANCH" "$CIRCLE_REPOSITORY_URL" . --depth 1
+      - checkout
```
Comment (mschile) [inline .circleci/workflows.yml:1345]: Is this slower now that we aren't doing `--depth 1`?

**Review by mschile — APPROVED** [review]

File: .circleci/workflows.yml (hunk 1 above)
Comment (emilyrohrbough) [reply .circleci/workflows.yml:1345]: yes it will be a bit slower

Comment (mschile) [reply .circleci/workflows.yml:1345]: Why not just leave it then?

Comment (emilyrohrbough) [reply .circleci/workflows.yml:1345]: Issues with 3rd-party contributors & the release-readiness step due to the git-history not being present. I want to take a clean look at it without it impacting the release.

**Pull request merged**